omc3/model/model_creators/lhc_model_creator.py

```diff
@@ -87,10 +87,10 @@ def get_correction_check_script(
     def prepare_run(cls, accel: Lhc) -> None:
         if accel.year in ["2018", "2022"]:  # these years should be handled by the fetcher
             symlink_dst = Path(accel.model_dir)/LHC_REPOSITORY_NAME
-            LOGGER.debug(f"Symlink destination: {symlink_dst}")
-            if symlink_dst.is_symlink():
-                symlink_dst.unlink()
-            symlink_dst.absolute().symlink_to((ACCELERATOR_MODEL_REPOSITORY/f"{accel.year}"))
+            if symlink_dst.exists() is False:
```

Comment:
just as comment, not that I care much: this should be `if not symlink_dst.exists()`

Reply:
This checks for a symlink and also for a normal directory in case you have actually copied the directory to that location.

Reply:
what andreas means is this: 
![image](https://user-images.githubusercontent.com/26184899/135276715-d5b9d2bc-f348-4de2-b4d3-6fa6bd699b20.png)
(https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations)

Reply:
yes, that's what I meant.
in the case "test if false" you don't really lose with this, since a `not` operation is needed anyway. but it looks odd

Reply:
I see! I fixed it in the code now but seems that I need two people to approve again!